static/index.html

```diff
@@ -6,13 +6,26 @@
   <title>Matoconv</title>
   <script>
 
-    function send()  {
-      var me_blob = new Blob([$('#content').val()], {type: 'text/html'});
-      var req = new XMLHttpRequest();
+    function send(send_type)  {
+      let me_blob;
+      let content_type = 'text/html';
+      let file_name = 'test.html';
+      let file_data = $('#content').val();
+
+      if (send_type == 'file') {
+        var let = document.getElementById('upload_file').files[0];
```

Comment:
`var let` has now gotten rid of the `file` variable definition. whoops.

Reply:
May be worth sticking `"use strict";` at the start of the script content. Modern browsers will highlight some of these mistakes then in the console.